### ISSUE TITLE
liquibaseDiffChangelog NoSuchMethodError .addForeignKeyColumn 

### ISSUE THREAD

**Issue by svict4** [liquibase/liquibase-hibernate#142]
Hi folks, I'm running the `liquibaseDiffChangelog` command on a jhipster project and am getting the below stacktrace.  `hibernate5/3.6` with `liquibase 3.5.3`
Looks like a similar issue to #117 

```
15:33:49.194 [main] ERROR liquibase - liquibase.structure.core.ForeignKey.addForeignKeyColumn(Lliquibase/structure/core/Column;)Lliquibase/structure/core/ForeignKey;

java.lang.NoSuchMethodError: liquibase.structure.core.ForeignKey.addForeignKeyColumn(Lliquibase/structure/core/Column;)Lliquibase/structure/core/ForeignKey;
        at liquibase.ext.hibernate.snapshot.ForeignKeySnapshotGenerator.addTo(ForeignKeySnapshotGenerator.java:57)
        at liquibase.ext.hibernate.snapshot.HibernateSnapshotGenerator.snapshot(HibernateSnapshotGenerator.java:84)
        at liquibase.snapshot.SnapshotGeneratorChain.snapshot(SnapshotGeneratorChain.java:50)
        at liquibase.ext.hibernate.snapshot.HibernateSnapshotGenerator.snapshot(HibernateSnapshotGenerator.java:76)
        at liquibase.snapshot.SnapshotGeneratorChain.snapshot(SnapshotGeneratorChain.java:50)
        at liquibase.snapshot.DatabaseSnapshot.include(DatabaseSnapshot.java:194)
        at liquibase.snapshot.DatabaseSnapshot.replaceObject(DatabaseSnapshot.java:292)
        at liquibase.snapshot.DatabaseSnapshot.replaceObject(DatabaseSnapshot.java:314)
        at liquibase.snapshot.DatabaseSnapshot.includeNestedObjects(DatabaseSnapshot.java:234)
        at liquibase.snapshot.DatabaseSnapshot.include(DatabaseSnapshot.java:208)
        at liquibase.snapshot.DatabaseSnapshot.init(DatabaseSnapshot.java:70)
        at liquibase.snapshot.DatabaseSnapshot.<init>(DatabaseSnapshot.java:44)
        at liquibase.snapshot.JdbcDatabaseSnapshot.<init>(JdbcDatabaseSnapshot.java:21)
        at liquibase.snapshot.SnapshotGeneratorFactory.createSnapshot(SnapshotGeneratorFactory.java:150)
        at liquibase.snapshot.SnapshotGeneratorFactory.createSnapshot(SnapshotGeneratorFactory.java:139)
        at liquibase.command.DiffCommand.createReferenceSnapshot(DiffCommand.java:190)
        at liquibase.command.DiffCommand.createDiffResult(DiffCommand.java:140)
        at liquibase.command.DiffToChangeLogCommand.run(DiffToChangeLogCommand.java:51)
        at liquibase.command.AbstractCommand.execute(AbstractCommand.java:8)
        at liquibase.integration.commandline.CommandLineUtils.doDiffToChangeLog(CommandLineUtils.java:209)
        at liquibase.integration.commandline.Main.doMigration(Main.java:967)
        at liquibase.integration.commandline.Main.run(Main.java:180)
        at liquibase.integration.commandline.Main.main(Main.java:99)
:liquibaseDiffChangelog FAILED
```

Can anyone shed any light?

┆Issue is synchronized with this [Jira Bug](https://datical.atlassian.net/browse/LB-451) by [Unito](https://www.unito.io/learn-more)

**Comment by kamenov** [comment]
Had the same issue and solved it by reverting from `liquibase:3.6.2` to `liquibase:3.5.2` while still using `hibernate5/3.6`.

**Comment by filipelautert** [comment]
This has been fixed in subsequent releases.
